FluentScreenRecorder/Views/MainPage.xaml

```diff
@@ -226,29 +226,30 @@
                 x:Name="BasicGridView"
                 Padding="12"
                 HorizontalAlignment="Stretch"
+                RightTapped="BasicGridView_RightTapped"
```

Comment:
I recommend that you handle ContextRequested instead as it also works with keyboard menu key (RightTapped only works when you right click with a mouse or long progress with touch).